### DISCUSSION TITLE
Topolvm support for `ppc64le` architecture

### DISCUSSION THREAD

**Discussion by Madhan-SWE** [topolvm/topolvm#611] [Q&A] [answered]
Hi Team,

I prepared topolvm images for `ppc64le` architecture from our Platform.
I deployed the helm chart version 0.15.3 on single node k8s cluster based on `ppc64le` architecture.
The installation was successful and I could run the sample application with the help of topolvm.

Can we expect support for `ppc64le` architecture from topolvm?

**Comment by toshipp** [comment, marked as answer]
We can not provide full support for ppc64le architecture because we have no hardware and our CI platform (github actions) also does not. By the way, we currently struggle to support ARM architecture in a limited form, ensuring build success and providing container images (https://github.com/topolvm/topolvm/pull/600). After that, it is possible to add pcc64le to the supported architecture similar to ARM.
